src/screenshotter.ts

```diff
@@ -30,13 +30,7 @@ export class Screenshotter {
 
   constructor(page: Page) {
     this._page = page;
-
-    const browserContext = page.context();
-    this._queue = (browserContext as any)[taskQueueSymbol];
-    if (!this._queue) {
-      this._queue = new TaskQueue();
-      (browserContext as any)[taskQueueSymbol] = this._queue;
-    }
+    this._queue = new TaskQueue();
```

Comment:
I'd actually understand if the queue went to the higher (browser) level, but it instead went to a page. Why?